espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/SourceDocument.java

```diff
@@ -75,18 +76,17 @@ private void buildXML() {
      * @param view Android View that will map to an Element
      */
     private void buildXML(Document doc, Element parentElement, View view) {
-        // TODO: Investigate if this is safe and if we need to strip out characters before constructing XML document
-        Element element = doc.createElement(view.getClass().getName());
+        Element element = doc.createElement(simpleClassName(view.getClass().getName()));
 
         // Set attributes
         ViewElement viewElement = new ViewElement(view);
-        element.setAttribute(ViewAttributesEnum.CONTENT_DESC.getName(), viewElement.getContentDescription().toString());
-        element.setAttribute(ViewAttributesEnum.BOUNDS.getName(), viewElement.getBounds().toShortString());
-        element.setAttribute(ViewAttributesEnum.FOCUSED.getName(), Boolean.toString(viewElement.isFocused()));
-        element.setAttribute(ViewAttributesEnum.CLICKABLE.getName(), Boolean.toString(viewElement.isClickable()));
-        element.setAttribute(ViewAttributesEnum.LONG_CLICKABLE.getName(), Boolean.toString(viewElement.isLongClickable()));
-        element.setAttribute(ViewAttributesEnum.CLASS.getName(), viewElement.getClassName());
-        element.setAttribute(ViewAttributesEnum.INDEX.getName(), Integer.toString(viewElement.getIndex()));
+        setAttribute(element, ViewAttributesEnum.CONTENT_DESC.getName(), viewElement.getContentDescription());
```

Comment:
you could overload the toString() method of ViewAttributesEnum to return the same value as getName. So then it will not be necessary to call getName explicitly here.